athenet/layers/layer.py

```diff
@@ -110,3 +115,17 @@ def b(self):
     @b.setter
     def b(self, value):
         self.b_shared.set_value(value)
+
+    def alloc_velocity(self):
```

Comment:
I would put some additional comment to emphasize importance of this function. One must understand that he must use free after it. Possible memory problems!